src/flags.c

```diff
@@ -215,8 +221,10 @@ char *maskname(int x)
     i += my_strcpy(s + i, "wallops, ");
   if ((x & LOG_SRVOUT) && raw_log)
     i += my_strcpy(s + i, "server output, ");
-  if ((x & LOG_BOTNET) && raw_log)
-    i += my_strcpy(s + i, "botnet traffic, ");
+  if ((x & LOG_BOTNETIN) && raw_log)
+    i += my_strcpy(s + i, "botnet traffic in, ");
```

Comment:
how about "botnet input"/"botnet output"? or "botnet incoming"/"botnet outgoing"? "botnet traffic in" sounds a bit odd IMHO

Reply:
Concur... botnet incoming/outgoing sounds best to me.